substrate/frame/assets/src/tests.rs

```diff
@@ -1775,3 +1775,336 @@ fn asset_destroy_refund_existence_deposit() {
 		assert_eq!(Balances::reserved_balance(&admin), 0);
 	});
 }
+
+mod sets {
```

Comment:
I'm not a fan of extremely long files, maybe we can move this to its own file under `tests/sets.rs`? What do you think?

Reply:
moved